quiz.rb

```diff
@@ -14,24 +14,32 @@
 # Take a string and return an array of normalized words.
 # normalized words contain no punctuation and are all lower case
 def normalize_words words
+  words.gsub(/[!@%&"]/,'').split.downcase
```

Comment:
order matters here. if you `split` before you `downcase`, you will get a `no method` error because an array does not have `downcase` as a method. when chaining methods, make sure that the return value of the previous method has access to the next method in the chain